app/models/miq_ae_field.rb

```diff
@@ -7,7 +7,7 @@ class MiqAeField < ApplicationRecord
   has_many   :ae_values,  :class_name => "MiqAeValue",  :foreign_key => :field_id, :dependent => :destroy,
                           :inverse_of => :ae_field
 
-  validates_uniqueness_of :name, :case_sensitive => false, :scope => [:class_id, :method_id]
+  validates :name, :uniqueness => {:scope => [:class_id, :method_id], :case_sensitive => false}, :if => :changed?
```

Comment:
`:if => name_changed?` please (unless name is derived from something else

if you want deletes, the next line could get merged up there

but that feels like a "while you are at it" and should probably be ignored

Reply:
aah, don't merge presence with this line

then it will never check presence